test/TestUtil.js

```diff
@@ -53,4 +53,11 @@ TestUtil.testSubsteps = function (fn, exprString, outputList,
   });
 };
 
+// Remove some property used in mathjs that we don't need and prevents node
+// equality checks from passing
+TestUtil.removeComments = function(node) {
+  node.filter(node => node.comment !== undefined).forEach(
+    node => delete node.comment);
+};
```

Comment:
oh interesting is this a new mathjs thing? I think there's something else we remove somewhere too (unless this is it)